README.md

```diff
@@ -2,7 +2,21 @@
 
 A best-effort reimplementation of the Minecraft Forge API.
 
-## Contribution Notes
+## How do I use this?
 
-We use [MinecraftForge @ d28cd0352b6b0fe86e062f29e681c3b14572c6d5](https://github.com/MinecraftForge/MinecraftForge/tree/d28cd0352b6b0fe86e062f29e681c3b14572c6d5) as our reference commit. All features should be based on this version of Forge
-for consistency. This will be changed once we update to 1.15.
+Patchwork is not yet ready for general use. However, if you're ambitious and want to test, we provide prebuilt releases on [GitHub](https://github.com/PatchworkMC/patchwork-api/releases). Note that you will need to pass any Forge mods through [Patchwork Patcher](https://github.com/PatchworkMC/patchwork-patcher/releases) as well. While we do not expect most mods to work, bug reports would be appreciated.
+
+If you know Java and would like to contribute, check out our [contribution guide](CONTRIBUTING.md)!
+
+
+## Technical details
+
+Patchwork API aims to re-implement the entirety of Minecraft Forge as a set of small and modular Fabric mods. This is in order to run Minecraft Forge mods patched using [Patchwork Patcher](https://github.com/PatchworkMC/patchwork-patcher) on Fabric.
+
+However, Patchwork aims not only to simply *run* Forge mods on Fabric but additionally aims to maintain a superior level of quality to Minecraft Forge. The Fabric toolchain is superior to Forge on a technical level, but lacks both the sheer quantity of hooks and features as well as existing mod support. Thus, Patchwork seeks to provide both the benefits of Fabric and the benefits of Forge.
```

Comment:
I know none of us like Forge but let's not assert that fabric is always better. We both know there are cases where it isn't
```suggestion
However, Patchwork aims not only to simply *run* Forge mods on Fabric but additionally aims to maintain a superior level of code quality to Forge. We feel the Fabric toolchain is superior to Forge on a technical level, but lacks both the sheer quantity of hooks and features as well as existing mod support. Thus, Patchwork seeks to provide both the benefits of Fabric and the benefits of Forge.
```